src/markup/data/auth.json

```
@@ -0,0 +1,14 @@
+[
+  {
+    "modify": "stroke",
+    "type": "text",
+    "label": "Логин",
+    "error": "Не верный логин"
```

В таких местах лучше тоже ставить `,` - это положительно сказывается на диффах в системах контроля версий
```suggestion
    "error": "Не верный логин",
```